nestkernel/source_table.cpp

```diff
@@ -296,21 +352,9 @@ nest::SourceTable::get_next_target_data( const thread tid,
     const Source& const_current_source =
```

Comment:
I would change the name to `current_source`, no need to have type information in the variable name. And I see no reason why to have a `const_current_source` here and then a `current_source` below.

Reply:
because we lift the restriction of thread-private data structures the idea was to use `const` to tell the compiler that only read access will follow. might be a matter of premature optimization though, i don't have a strong opinion on it.

Reply:
For just the single use of the "const" version (on line 377 below), I think the duplicity of a const and a plain version are not worth the extra code complexity.